content/kubermatic/master/tutorials-howtos/external-clusters/_index.en.md

```diff
@@ -1,55 +1,63 @@
 +++
-title = "Adding an External Kubernetes Cluster"
+title = "External Kubernetes Clusters"
 date = 2022-01-10T14:07:15+02:00
 weight = 7
 
 +++
 
-This section describes how to add and manage existing Kubernetes clusters known as external clusters in KKP.
-You can import or connect a cluster.
-- Import: You can import a cluster via credentials. Imported Cluster can be viewed and edited i.e, upgrade the control plane version or scale the nodes. Currently, GKE, AKS, and EKS clusters are supported.
-- Connect: You can also connect any other clusters in the KKP via kubeconfig. Connected clusters can only be viewed, not edited.
+This section describes how to add, create and manage Kubernetes clusters known as external clusters in KKP.
+You can create a new cluster or import/connect an existing cluster.
+- Import: You can import a cluster via credentials. Imported Cluster can be viewed and edited. 
+  Supported Providers:
+  - Google Kubernetes Engine (GKE)
+  - Azure Kubernetes Service (AKS)
+  - Amazon Elastic Kubernetes Service (EKS) 
 
-The KKP platform uses existing kubeconfig or generates the new one from the cloud provider API.
+- Connect: You can also connect any cluster in the KKP via kubeconfig. Connected clusters can only be viewed, not edited.
+
+The KKP platform uses the provided kubeconfig or generates a new one from the cloud provider API.
 The KKP backend takes advantage of this kubeconfig to retrieve the cluster information, its' nodes, metrics, and events.
 Every cluster update is performed only by the cloud provider client. There is no need to install any agent on the cloud provider side.
 
 ## Prerequisites
 
 The following requirements must be met to add an external Kubernetes cluster:
- - The external Kubernetes cluster must already exist before you begin the import/connect process. Please refer to your cloud
- provider documentation for instructions.
+ - The external Kubernetes cluster must already exist before you begin the import/connect process. Please refer to the cloud provider documentation for instructions.
```

Comment:
Maybe you can change this sentence to be:
```
Unless you are creating the external cluster, it must already exist before you begin the import/connect process. Please refer to the cloud provider documentation for instructions.
```
or something similar. Because for the "create" feature the cluster won't exist in the cloud provider.